src/version.rs

```diff
@@ -54,15 +65,8 @@ impl Default for Version {
     }
 }
 
-impl fmt::Debug for Version {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        use self::Http::*;
-
-        f.write_str(match self.0 {
-            Http09 => "HTTP/0.9",
-            Http10 => "HTTP/1.0",
-            Http11 => "HTTP/1.1",
-            H2     => "HTTP/2.0",
-        })
+impl Display for Version {
```

Comment:
We specifically *didn't* implement `Display` because there isn't a canonical way to serialize a version. For instance, the HTTP2 identifier is "h2", but it's wrong to write `GET / h2`.

Reply:
👋 @seanmonstar! quick question - is this true for the `Response` type itself? That `Display` was intentionally _not_ implemented due to questions around what it would mean to implement for HTTP/2.0 responses?

I did a [quick impl](https://github.com/hyperium/http/compare/master...dadleyy:master) that formats the response for HTTP/1.0 correctly I think

Reply:
Yes, it's true for `Request` and `Response` as well.